src/d/alexa-openwebif/source/skill.d

```diff
@@ -190,3 +213,18 @@ unittest
 	skill = new OpenWebifSkill("", "en-US");
 	assert(skill.getText(TextId.PleaseLogin) == AlexaText_en[TextId.PleaseLogin].text);
 }
+
+AlexaResult returnError(ITextManager texts)
+{
+	import std.format : format;
+	import std.random : uniform;
+	import std.conv : to;
+
+	AlexaResult result;
+	auto errorId = uniform!uint();
```

Comment:
lets atleast present this id to the user as a hexstring for the sake of readability